app/assets/javascripts/components/student/student_card.es6.jsx

```diff
@@ -58,8 +58,8 @@ class StudentCard extends Component {
         return <StudentContact student={this.props.student} />;
       case 'conference':
         return <StudentConference student={this.props.student} />;
-      case 'parent':
-        return <StudentParent student={this.props.student} />;
+      case 'guardian':
```

Comment:
Can you refactor all the string literals here to use the `TypeConstants`? You'll also need to change the `parent` constant to `guardian` and then fix this for components that use the `parent` constant!